src/app/app.routes.ts

```diff
@@ -0,0 +1,3 @@
+import { Routes } from '@angular/router';
+
+export const routes: Routes = [];
```

Comment:
document

          export const routes: Routes = []; // Initialize an empty routes array

          Explantion: Adding a comment here indicates that this is where routes will be defined, even though it's currently empty.